docker/metrics-collector/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3.1'
+
```

Comment:
maybe you considered this, but anyway:

if you will add this configuration to prometheus compose file:

```
networks:
  default:
    external:
      name: wnodetestcluster_cluster
```

it will be possible to use as a targets
 
```
targets: ['wnodetestcluster_wnode_3:8545', 'wnodetestcluster_wnode_2:8545', 'wnodetestcluster_wnode_1:8545']
```

thus making configuration usable not only on macos

Reply:
I haven't tested that actually! I will try this approach. If this works, then the only problem will be a dynamic number of `wnode` nodes.